lib/vanagon/platform/windows.rb

```diff
@@ -37,6 +37,25 @@ def package_name(project)
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_packaging_artifacts(workdir, name, binding)
+        # HOW DO I WORK OUT the Package Type Here - Can I use the binding as per next statement ??
+        # Or should we be splitting out this into a nuget and a windows sub-class - i.e. too many package_type
+        package_type = binding.eval("@platform.package_type")
+        case package_type
```

Comment:
ummmmm, check out the `package_name` and `generate_package` methods above.... Lines 24 and 9 (`case project.platform.package_type`) have what you need. Notice it's contained in the project object, so you'll have to pass that in to access it.

Might I suggest not conditionally generating package artifacts. Generate all of them regardless. It doesn't take a lot of time to process templates, and they don't do any harm if we just leave them hanging around. We have the ability to only move in the ones we want when building the package in question.

i.e.,
```
"#{@copy} #{project.name}.nuspec $(tempdir)/#{project.name}/",
"#{@copy} chocolateyInstall.ps1 chocolateyUninstall.ps1 $(tempdir)/#{project.name}/tools/",
```